pkg/agent/config/node_config.go

```diff
@@ -45,18 +45,25 @@ const (
 )
 
 var (
-	// VirtualServiceIPv4 / VirtualServiceIPv6 are used in the following situations:
-	// - Use the virtual IP to perform SNAT for packets of Service from Antrea gateway and the Endpoint is not on
-	//   local Pod CIDR or any remote Pod CIDRs. It is used in OVS flow of table serviceConntrackCommitTable.
-	// - Use the virtual IP to perform DNAT for packets of NodePort on host. It is used in iptables rules on host.
-	// - Use the virtual IP as onlink routing entry gateway in host routing entry.
-	// - Use the virtual IP as destination IP in host routing entry. It is used to forward DNATed NodePort packets
-	//   or replied SNATed Service packets back to Antrea gateway.
-	// - Use the virtual IP for InternalIPAddress parameter of Add-NetNatStaticMapping.
+	// VirtualGWIPv4 or VirtualGWIPv6 is used as onlink routing entry gateway in host routing entry.
+	VirtualGWIPv4 = net.ParseIP("169.254.0.254")
+	VirtualGWIPv6 = net.ParseIP("fc01::aabb:ccdd:eeff")
```

Comment:
I still don't get why we need another gateway IP as next hop instead of making SNAT IP and DNAT IP as onlink addresses directly. If we have just DNAT IP and SNAT IP, DNAT IP represents OVS as a receiver, host network forwards ingress traffic to this IP when it's supposed to be handled by OVS, and SNAT IP represents OVS as a sender, host network receives traffic from this IP when OVS has to masquerade certain traffic.

Reply:
Only added a virtual DNAT IP now.